Kernel/Net/NetworkAdapter.h

```diff
@@ -120,4 +122,14 @@ class NetworkAdapter
     u32 m_mtu { 1500 };
 };
 
+template<typename... Parameters>
+void dmesgln_net(NetworkAdapter const& device, AK::CheckedFormatString<Parameters...>&& fmt, Parameters const&... parameters)
+{
+    AK::StringBuilder builder;
+    MUST(builder.try_append(device.class_name()));
```

Comment:
Logging should not crash the system on out of memory, please use a different pattern.

Reply:
To clarify - the `try_append` calls are not OK here?

Reply:
The MUST macros assert success. If any of this fails the kernel will assert.

Reply:
I have reworked the code to no longer use `MUST`.